Milestone 3 react calculator app

kalavhan opened the pull request:
Pull request for the review of the third milestone of the Calculator react app
Changes added to this pull request:

- Width of the calculator set to 700px
- height of display and rows of the button set to 100px
- Add wide and color props to the button component
- color by default orange for buttons
- 25% of width by default to the buttons

Oghenebrume50 commented on src/index.css:
```diff
@@ -1,3 +1,53 @@
+  flex-direction: column;
+}
+
+.display {
```
This should use flex display 
![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)

kalavhan commented:
@Oghenebrume50 Thanks a lot for the review, I pushed the requested changes:
.calculator is the class for the app
`.calculator {
  width: 700px;
  display: flex;
  flex-direction: column;
}`

.display was missing the display:flex property, now added
`.display {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  text-align: right;
  background-color: #808080;
  height: 100px;
  color: #fff;
  font-weight: 900;
  font-size: 80px;
}`

.buttonPanel was missing the display:flex property, now added
`.buttonPanel {
  display: flex;
  flex-direction: column;
}`

shshamim63 commented on src/components/ButtonPanel.js:
```diff
@@ -0,0 +1,91 @@
+      />
+      <Button
+        name="6"
+        color="light-gray"
+      />
+      <Button
+        name="-"
+      />
+    </div>
+    <div className="buttonPanel-row">
+      <Button
+        name="1"
+        color="light-gray"
+      />
+      <Button
+        name="2"
+        color="light-gray"
+      />
+      <Button
+        name="3"
+        color="light-gray"
+      />
+      <Button
+        name="+"
+      />
+    </div>
+    <div className="buttonPanel-row">
+      <Button
+        name="0"
+        color="light-gray"
+        wide
+      />
+      <Button
+        name="."
+        color="light-gray"
+      />
+      <Button
+        name="="
+      />
+    </div>
```
[Suggestion]
This block of code is long and contains a lot of repetition. You can refactor it. How? Well, I can give you a hint. Think about an array contains arrays that contain a group of buttons symbol. Then you can iterate it the array.
